vulnerabilities/models.py

```diff
@@ -199,3 +200,25 @@ def make_data_source(self, batch_size: int, cutoff_date: datetime = None) -> Dat
 
     def __str__(self):
         return self.name
+
+
+class VulnerabilitySeverity(models.Model):
+
+    scoring_system_choices = ((system.identifier, system.name) for system in scoring_systems.values())  # nopep8
+    vulnerability = models.ForeignKey(Vulnerability, on_delete=models.CASCADE)
+    value = models.CharField(max_length=50, help_text="Example: 9.0, Important, High")
+    scoring_system = models.CharField(
+        max_length=50,
+        choices=scoring_system_choices,
+        help_text="Identifier for the scoring system used. Avaiable choices are: {} ".format(
```

Comment:
```suggestion
        help_text="Identifier for the scoring system used. Available choices are: {} ".format(
```
Also... it may be easier to read if you put the join below in a variable.